ui/component/wunderbarSuggestions/view.jsx

```diff
@@ -228,12 +229,19 @@ export default function WunderBarSuggestions(props: Props) {
     function handleKeyDown(event) {
       const { ctrlKey, metaKey, keyCode } = event;
 
+      if (keyCode === K_KEY_CODE && ctrlKey) {
```

Comment:
I'm not certain that the next block `if (!inputRef.current) return;` is necessary, but if it is, I imagine it should be before this one. (Bail early?)

Reply:
You are right!  I thought that line was guarding against when the wunderbar is not selected but it is actually just checking that element exists.  I will make this change.